_drafts/SC2-4-4-link-has-name.md

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
In the Applicability I suggest making two changes:
- are --> is
- having "semantic role" link to a definition and delete the Note.

"The rule applies to any HTML or SVG element with the semantic role of link that are ..." + **Note**:  
--> "The rule applies to any HTML or SVG element with the [semantic role][] of link that is ..." (+ no note)